### PR TITLE
update glossary shortcodes

### PR DESCRIPTION
**Pull request by nadinequinr3**
- Improved usability by removing the `definition` tooltip.
- The term passed to the `tooltip` shortcode is now case-insensitive and simple plural versions match. 
- Removed all but one of the calls to the `tooltip` shortcode in advance of another PR to apply the tooltip in a consistent manner across all sections. (One call left for testing purposes.)

### CODE REVIEW

File: content/en/platform/corda/5.0/application-networks/creating/_index.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ menu:
 section_menu: corda5
 ---
 # Creating Application Networks
-This section describes how to set up a dynamic network. The network requires a running {{< tooltip >}}MGM{{< definition term="MGM" >}}{{< /tooltip >}} that all members must register with before they can transact among the group. This section also describes how to onboard a new member as a {{< tooltip >}}notary{{< definition term="Notary" >}}{{< /tooltip >}} service representative. 
+This section describes how to set up a dynamic network. The network requires a running {{< tooltip >}}MGM{{< /tooltip >}} that all members must register with before they can transact among the group. This section also describes how to onboard a new member as a notary service representative. 
 
 The main sections describe a standard configuration but you can learn more about using mutual TLS or session certificates in the [Optional Configurations]({{< relref"./optional/_index.md">}}) section.
-
+
```
Comment (yarivus) [inline content/en/platform/corda/5.0/application-networks/creating/_index.md:17]: redundant spaces

File: content/en/platform/corda/5.0/application-networks/creating/_index.md
Comment (yarivus) [inline content/en/platform/corda/5.0/application-networks/creating/_index.md:1]: I see that you removed tooltips ... 
Assuming they will return once the tooltip shortcode is better established.

**Review by yarivus — COMMENTED** [review]
checkpoint review

File: content/en/platform/corda/5.0/application-networks/creating/_index.md
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/application-networks/creating/_index.md:1]: You were too quick - I was adding a comment to the PR.

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/application-networks/creating/_index.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ menu:
...
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/application-networks/creating/_index.md:17]: I deleted the spaces but the line break is intentional.

File: content/en/platform/corda/5.0/application-networks/creating/_index.md
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/application-networks/creating/_index.md:1]: We're going to apply tooltips for all glossary terms throughout the docs in another branch. This PR was to ensure the shortcodes were defined correctly before applying them. 
I'd rather start from scratch so it's done in a consistent way. I don't want bulk searches to return a mixture of instances with and without tooltips. We also won't necessarily want every single occurrence of a term to have a tooltip so perhaps some of the existing ones won't be required.

Comment (yarivus) [reply content/en/platform/corda/5.0/application-networks/creating/_index.md:1]: Understood

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**